scripts/bootstrap/bootstrap

```diff
@@ -61,17 +61,37 @@ exists-non-empty() {
     [[ -d $1 ]] && files="$(ls -A -- "$1")" && [[ -n $files ]]
 }
 
+already-installed-message() {
+    log "Toltec is already installed or partially installed"
+    log "Toltec is already installed or partially installed"
```

Comment:
Duplicate?

Reply:
Hmm, something must have gone wrong when I copy pasted it.